docker/docker-compose.yml

```diff
@@ -121,18 +130,21 @@ services:
         source: build
         target: /app/build/
     environment:
-      ENVIRONMENT: ${ENVIRONMENT:-localhost}
-      WEBSOCKET_PORT: 8080
+      AUTOSTART_RETRIES: ${AUTOSTART_RETRIES:-1000}
+      BLOCKCHAIN_URL: ${BLOCKCHAIN_URL}
       BLOCKCHAIN_WS_HOST: ${BLOCKCHAIN_WS_HOST:-blockchain}
       BLOCKCHAIN_PORT: ${BLOCKCHAIN_PORT:-8546}
-      MONGO_URL: ${OPTIMIST_MONGO_URL:-mongodb://mongodb:27017}
+      ENVIRONMENT: ${ENVIRONMENT:-localhost}
       HASH_TYPE: poseidon
+      IS_CHALLENGER: ${IS_CHALLENGER:-true}
       LOG_LEVEL: ${LOG_LEVEL:-debug}
       LOG_HTTP_PAYLOAD_ENABLED: ${LOG_HTTP_PAYLOAD_ENABLED:-true}
       LOG_HTTP_FULL_DATA: ${LOG_HTTP_FULL_DATA:-false}
-      IS_CHALLENGER: ${IS_CHALLENGER:-true}
-      PROPOSER_MAX_BLOCK_PERIOD_MILIS: ${PROPOSER_MAX_BLOCK_PERIOD_MILIS:-0}
       MAX_BLOCK_SIZE: ${MAX_BLOCK_SIZE:-50000}
+      MONGO_URL: ${OPTIMIST_MONGO_URL:-mongodb://mongodb:27017}
+      PROPOSER_MAX_BLOCK_PERIOD_MILIS: ${PROPOSER_MAX_BLOCK_PERIOD_MILIS:-0}
+      USE_EXTERNAL_NODE: ${USE_EXTERNAL_NODE}
+      WEBSOCKET_PORT: 8080
```

Comment:
We could always pass some variable in parameters or default value as a normal rule in docker compose files.
```
WEBSOCKET_PORT: ${WEBSOCKET_PORT:8080}
```

Reply:
Good catch, this was a copy/paste thing.